apps/website/lib/trpc/react.tsx

```
@@ -0,0 +1,68 @@
+import { getBaseUrl } from '@cuhacking/utils'
+
+import { api } from './trpc'
+
+const createQueryClient = () => new QueryClient()
+
+let clientQueryClientSingleton: QueryClient | undefined
+function getQueryClient() {
+  if (typeof window === 'undefined') {
+    return createQueryClient()
+  }
+  return (clientQueryClientSingleton ??= createQueryClient())
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)